### PR TITLE
M0078DDG-480 Improve Sandboxed Application documentation

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
Preview: https://docs.microej.com/en/feature-m0078ddg-480_sandboxed-application/

This PR aims to improve the documentation on Sandboxed Application development.

### CODE REVIEW

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+=================
+
+A Sandboxed Application can be deployed on a device in several ways:
+
+- Local Deployment: the device is connected to the developer's computer and the SDK transfers the Application on it.
+- Remote Deployment: the device connects to a Repository where the Application is stored, and deploys it over the network.
+
+In both cases, the Multi-Sandbox Firmware must be previously flashed on the device.
+Please refer to the :ref:`kernel-developer-guide` to learn how to do it and to 
+the `Resources Repository <https://repository.microej.com/index.php?resource=FIRM&topic=ALL&version=ANY&edition=ANY>`_
```
Comment (tdelhomenie-microej) [inline ApplicationDeveloperGuide/sandboxedAppDeploy.rst:13]: @frivieremicroej is it ok to refer to this URL for the list of available firmwares ?

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    BUILD SUCCESSFUL
+
+    Total time: 1 minute 6 seconds
+
+The WPK file produced by the build process is located in a dedicated
+``target~/artifacts`` folder in the project and is published to the 
+target module repository declared in :ref:`MicroEJ Module Manager settings file <mmm_settings_file>`.
+
+The module repository can be a `MicroEJ Forge instance <https://www.microej.com/product/forge/>`_.
+
```
Comment (tdelhomenie-microej) [inline ApplicationDeveloperGuide/sandboxedAppDeploy.rst:94]: @frivieremicroej I used the same content than in the previous doc for Remote Deployment but I don't think it is sufficient. It does not explain what is Forge and how to install and use. Don't we have such a doc somewhere?

File: ApplicationDeveloperGuide/sandboxedAppSim.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
+.. _chapter.application.sim:
+
+Run on the Simulator
+====================
+
+Once :ref:`a Sandboxed Application project has been created <chapter.application.firstApplication>`, it can be tested on the Simulator.
+
+The Simulator requires a Virtual Device to execute the Application.
+Please refer to the :ref:`kernel-developer-guide` to learn to get or create one.
```
Comment (tdelhomenie-microej) [inline ApplicationDeveloperGuide/sandboxedAppSim.rst:9]: Don't we have some Evaluation VD to link to? Should we redirect to the list of platforms (https://repository.microej.com/index.php?resource=JPF&topic=ALL&version=ANY&edition=ANY) ?

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst (hunk 1 above)
Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst:13]: OK, event almost all firmware are 4 years old (the last published one has 2 years)

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst (hunk 2 above)
Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst:94]: no, this is the only entry point for Forge. OK for me.

File: ApplicationDeveloperGuide/sandboxedAppSim.rst (hunk 3 above)
Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppSim.rst:9]: We can point to https://developer.microej.com/create-your-first-application/get-started-app-development-on-espressif-esp32-wroverkit-v41/ VD (https://repository.microej.com/packages/black/1.1.1/vd/ESP32-WROVER/BLACK-ESP32WROVER-RQQAW-1.1.1.vde)

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- duplicate the Run Configuration created in the chapter :ref:`chapter.application.sim`,
+
+.. figure:: images/sandboxed-application-duplicate-run-conf.png
+   :alt: Duplicate Run Configuration
+   :align: center
+
+   Duplicate Run Configuration
+
+- rename the duplicated Run Configuration, for example by prefixing by ``(Local)``,
+- in the ``Execution`` tab, modify the ``Execution`` mode to ``Execute on Device``,
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/sandboxedAppDeploy.rst:35]: use GUILabels (check everywhere)

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst (hunk 2 above)
Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst:94]: please contact our support team if you want more information on Forge and automated Sandboxed Applications deployment through your device management system.

Comment (frivieremicroej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst:94]: please review & merge https://github.com/MicroEJ/docs/pull/435

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst:94]: Review done

File: ApplicationDeveloperGuide/sandboxedAppDeploy.rst (hunk 4 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/sandboxedAppDeploy.rst]: Done

**Comment by frivieremicroej** [comment]
Replaced by https://github.com/MicroEJ/docs/pull/435

**Pull request closed without merging**